### ISSUE TITLE
Conflict with Ajax Load More plugin

### ISSUE THREAD

**Issue by MrCarlLister** [GM-Alex/user-access-manager#99]
For some reason when User Access Manager plugin is active, the Ajax Load more plugin (https://en-gb.wordpress.org/plugins/ajax-load-more/) won't load additional content into the page.

It works when I'm logged in as admin, but if I'm not logged in as admin the additional content won't load.

I know it's difficult when there's a conflict between two plugins but my gut says that there's something at this end that is stopping ALM working and not vice versa.

In the meantime I am going to try manual usage of ALM as opposed to plugin but thought I would flag here.

Edit: I've checked and yes, if a user is logged in and has a user group associated with them or is admin the content loads correctly. Strangely, the content in question is not specific to a content group.

**Comment by MrCarlLister** [comment]
I've found the issue. Kind of.

Line 515 in AccessHandler.php

`$excludedPosts += $userGroup->getFullPosts();`

Commenting that line out will allow the ajax load more to include all custom posts. I don't have time to dig any deeper but I think there might be an issue with the writing permissions because if I set the writing permissions for the group to all it works also.

Hope that helps, in the meantime I'll look at a non-destructive work around.
